qless/__init__.py

```diff
@@ -129,6 +132,8 @@ def __init__(self, url='redis://localhost:6379', hostname=None, **kwargs):
         import socket
         # This is our unique idenitifier as a worker
         self.worker_name = hostname or socket.gethostname()
+        if PY3:
+            kwargs['decode_responses'] = True
```

Comment:
@dlecocq: How do you feel about this change? This greatly simplifies the conversion to Python 3, but it restricts the ability to have raw binary data inside Job data.